api/handlers/abstractcontainerhandler.py

```diff
@@ -1,95 +1,64 @@
-from webob import exc
+from webapp2 import Request
 
-from ..web import base
 from .. import config
+from ..web import base
+from ..web.errors import APINotFoundException
+
+
+# Efficiently search in multiple collections
+CONTAINER_SEARCH_JS = r"""
+(function searchContainer(_id) {
+    if (/^[a-f\d]{24}$/i.test(_id)) {
+        _id = ObjectId(_id);
+    }
+    return {
+        "groups": db.getCollection("groups").findOne({"_id" : _id}, {"_id": 1}),
+        "projects": db.getCollection("projects").findOne({"_id" : _id}, {"_id": 1}),
+        "sessions": db.getCollection("sessions").findOne({"_id" : _id}, {"_id": 1}),
+        "acquisitions": db.getCollection("acquisitions").findOne({"_id" : _id}, {"_id": 1})
```

Comment:
We'll probably want to add analyses to this list as well

Reply:
Collections, Y/N ?